src/services/mentors.js

```diff
@@ -364,6 +364,8 @@ module.exports = class MentorsHelper {
 				...data,
 				...saasPolicyData,
 				visible_to_organizations: userOrgDetails.data.result.related_orgs,
+				visibility: organisationPolicy.mentee_visibility_policy,
```

Comment:
@adithyadinesh0412 Please check the key visibility. should it be changed? Make sure we replaced visibility keys usage from relevant places to avoid logic break. test org,user,mentor extension creation , updation flows also once